src/network/netdev/tunnel.c

```diff
@@ -617,6 +629,33 @@ int config_parse_encap_limit(const char* unit,
         return 0;
 }
 
+CONFIG_PARSER_PROTOTYPE(config_parse_6rd_prefix) {
+        Tunnel *t = userdata;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        union in_addr_union p;
+        uint8_t l;
+        int r;
+
+        r = in_addr_prefix_from_string(rvalue, AF_INET6, &p, &l);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "6rdPrefix \"%s\" is not a valid IPv6 prefix, ignoring", rvalue);
```

Comment:
Please add the error cause in the log message, that is `"..., ignoring: %m"`. So, I think the error message can be simply `"Failed to parse 6rd prefix '%s', ignoring: %m"`, as usually the error is `EINVAL`. This way, the error message is also valid for `ENOMEM` or other errors.

Reply:
Done.